titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/JobManagerUtil.java

```diff
@@ -148,6 +149,20 @@ public static Task attachPlacementData(Task task, Optional<TitusExecutorDetails>
         }).orElse(task);
     }
 
+    public static Task attachKubeletData(Task task, PodWrapper podWrapper) {
+        if(podWrapper.getV1Pod().getStatus() == null) { return task; }
+        String ipv4 = podWrapper.getV1Pod().getStatus().getPodIP();
+        if (ipv4 == null || ipv4.isEmpty()) {
+            return task;
+        } else {
```

Comment:
minor: remove the `else` (previous `if` clause already has an early `return`) and reduce indentation level